src/image_copy/main.go

```diff
@@ -67,21 +70,68 @@ func getTags(dstTag string, updateMajor bool) []string {
 	return tags
 }
 
+func verifyImage(imageTag, platform, check string) {
+	fmt.Printf("verifying binaries in '%s' from '%s' for platform '%s' are '%s'\n", binFolder, imageTag, platform, check)
+
+	cmd := exec.Command(
+		"docker",
+		"run",
+		"--platform", platform,
+		"--user=root",
+		"--rm",
+		"--entrypoint=sh",
+		imageTag,
+		"-c", `
+apk add -U file > /dev/null
+
+count=0
+for file in `+binFolder+`/*; do
+  if [ -f "$file" ]; then
+    count=$((count+1))
+    if file "$file" | grep -q "`+check+`"; then
+      echo "$file is `+check+`"
+    else
+      echo "$file is not `+check+`"
+      file "$file"
+      exit 1
+    fi
+  fi
+done
+
+if [ $count -lt 1 ]; then
+  echo "no binaries were found"
+  exit 1
+fi
+`)
+
+	var out bytes.Buffer
+	cmd.Stdout = &out
+	cmd.Stderr = &out
+	err := cmd.Run()
+	fmt.Println(out.String())
+	if err != nil {
+		log.Fatal(err.Error())
+	}
```

Comment:
```suggestion
	out, err := cmd.CombinedOutput()
	fmt.Println(string(out))
	if err != nil {
		log.Fatal(err.Error())
	}
```

CombinedOutput does all that for you